app/javascript/components/App.js

```diff
@@ -13,6 +13,36 @@ import SongShow from './pages/SongShow'
 import About from './pages/About'
 
 class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      songs: []
+    }
+  }
+  componentDidMount(){
+    this.readSong()
+  }
+
+  createSong = (newSong) => {
+    fetch("http://localhost:3000/songs", {
```

Comment:
Since all the code is coming from the same base url you can fetch to just your endpoint `/songs`